deploy/ocm-agent-operator-managednotifications/node-condition/100-sre-node-condition-managed-notification.ManagedNotification.yaml

```diff
@@ -15,6 +15,8 @@ spec:
       resolvedBody: |-
         DiskPressure condition is resolved.
       resendWait: 6
+      references: "https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/storage/configuring-persistent-storage"
```

Comment:
https://github.com/openshift/ocm-agent-operator/blob/master/api/v1alpha1/managednotification_types.go#L58C20-L58C20
The `references` should be a slice type? Have you tried to send a test SL with staging cluster and confirmed you can see the value there?